parent/pom.xml

```diff
@@ -545,7 +545,7 @@
         <plugin>
           <groupId>org.codehaus.plexus</groupId>
           <artifactId>plexus-component-metadata</artifactId>
-          <version>1.6</version>
+          <version>1.7</version>
```

Comment:
I'm really not sure why this is required as we haven't any change anything wrt to the build setup. But its fine for me.

Reply:
@rhuss y no idea. Somehow it would reference the old group name of 'jdom' instead of the new 'org.jdom', which is fixed in 1.7.

Reply:
ok, thanks.